ads/kixer.js

```diff
@@ -39,11 +44,34 @@ export function kixer(global, data) {
     d.removeEventListener('load', kxload, false);
     if (d.childNodes.length > 0) {
       global.context.renderStart();
+      view_interval = setInterval(function() {
+        kxview_check(); // Once the ad has loaded, start checking for an ad view
```

Comment:
Hi Lanka! Thanks for your comment. It makes sense, however, this helped to make sure the ad was in the viewport for at least 1 second and prevented duplicate/early calls to the callback function.